package.json

```diff
@@ -21,16 +22,15 @@
     "react-dom": "^16.2.0",
     "react-masonry-component": "^6.0.2"
   },
-  "keywords": [
-    "next"
-  ],
+  "keywords": ["next"],
   "license": "MIT",
   "main": "n/a",
   "scripts": {
-    "dev": "node server.js",
+    "dev": "./node_modules/.bin/nodemon --watch server.js server.js",
```

Comment:
You dont need ./node_modules/.bin npm or yarn automatically does that

Reply:
Oh nice, I assumed it would use the globally installed version of nodemon if that wasn't there.
This is supposed to use the locally installed one, if yarn does this automatically it can be removed.

Reply:
Yeah remove the relative part

Reply:
Changed, after merge though, so changes aren't in main repo. It's such a small change, so Jesse can do it on stream in like 3 seconds.